pkg/tf2pulumi/convert/testdata/builtins/pcl/main.pp

```diff
@@ -63,4 +66,5 @@
 }
 output funcSplit2 {
   value = split(",", "")
-}
+}#endif
```

Comment:
#endif introduced on its own?

Reply:
Ha it was writing the pragmas out to the HCL file, which mostly got dropped because we don't parse comments fully but this one somehow survived the translation process 😆 
Fixed it so the converted HCL doesn't have the pragma lines left in it.